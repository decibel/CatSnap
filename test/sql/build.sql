\set ECHO none

\i test/pgxntool/psql.sql

BEGIN;
\i sql/cat_snap.sql

\d cat_snap.entity
\d cat_snap.raw_type

\echo # TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab sw=2 ts=2
