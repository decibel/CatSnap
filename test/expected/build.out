\set ECHO none
 count 
-------
    33
(1 row)

         Table "_cat_snap.entity"
     Column     |    Type     | Modifiers 
----------------+-------------+-----------
 entity         | text        | not null
 entity_type    | entity_type | not null
 attributes     | attribute[] | not null
 delta_keys     | text[]      | 
 delta_counters | text[]      | 
 delta_fields   | text[]      | 
Indexes:
    "entity_pkey" PRIMARY KEY, btree (entity)

    Composite type "cat_snap.raw_type"
     Column     |     Type     | Modifiers 
----------------+--------------+-----------
 xmin           | xid          | 
 oid            | oid          | 
 typname        | text         | 
 typnamespace   | oid          | 
 typowner       | oid          | 
 typlen         | smallint     | 
 typbyval       | boolean      | 
 typtype        | "char"       | 
 typcategory    | "char"       | 
 typispreferred | boolean      | 
 typisdefined   | boolean      | 
 typdelim       | "char"       | 
 typrelid       | oid          | 
 typelem        | oid          | 
 typarray       | oid          | 
 typinput       | regproc      | 
 typoutput      | regproc      | 
 typreceive     | regproc      | 
 typsend        | regproc      | 
 typmodin       | regproc      | 
 typmodout      | regproc      | 
 typanalyze     | regproc      | 
 typalign       | "char"       | 
 typstorage     | "char"       | 
 typnotnull     | boolean      | 
 typbasetype    | oid          | 
 typtypmod      | integer      | 
 typndims       | integer      | 
 typcollation   | oid          | 
 typdefaultbin  | pg_node_tree | 
 typdefault     | text         | 
 typacl         | aclitem[]    | 

# TRANSACTION INTENTIONALLY LEFT OPEN!
